motoko/counter/README.md

```diff
@@ -21,6 +21,7 @@ The application provides an interface that exposes the following methods:
 ### Prerequisites
 This example requires an installation of:
 
+- [x] DFX version 0.19.0 or newer
```

Comment:
was this added to the wrong example?

Reply:
i.e. `motoko/counter` instead of `motoko/canister_logs`

Reply:
yes, thank you!